site/content/api/Tenant-API.md

```diff
@@ -135,15 +139,27 @@ The response message's *status* property may contain the following codes:
 | Code  | Description |
 | :---- | :---------- |
 | *204* | No Content, the tenant has been updated successfully. |
-| *404* | Not Found, there is no tenant identified with the given *tenant-id*. |
+| *404* | Not Found, there is no tenant identified with the given *tenant_id*. |
 
 For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
 
 ## Remove Tenant
 
-Clients use this command to *remove* all information about a tenant from Hono. Once a tenant is removed, clients can no longer use Hono to interact with any device of the tenant nor can they consume any more data produced by the devices of the tenant.
+Clients use this command to *remove* all information about a tenant from Hono.
+
+Once a tenant is removed:
```

Comment:
**If** a tenant is removed?
Would reflect better IMHO  that this might not be immediately considered by other components (as told below then)...

Same format (one header line with item list below) for update as well?
So the reader is better noticed that now some impacts on the operation follows that impact the overall system.